source/mainnet/net/guides/run-node.rst

```diff
@@ -28,66 +28,175 @@ Before running a Concordium node you will need to
 
    -  On *Linux*, allow Docker to be run as a non-root user.
 
-2. Download and extract the :ref:`full suite for running a Mainnet node on Linux<concordium-docker-package-download>` or :ref:`full suite for running a Testnet node on Linux<concordium-docker-package-download-testnet>`.
+2. `Install the compose plugin. <https://docs.docker.com/compose/install/>`_
 
 .. _running-a-node:
 
 Running/upgrading a node
 ========================
 
-To start running a client that will join Mainnet or Testnet follow these
-steps:
+Concordium provides two docker images, a
+`mainnet <https://hub.docker.com/r/concordium/mainnet-node>`_ one and a
```

Comment:
Should we add those links to "Downloads - Mainnet" and "Downloads - Testnet". We could make the Docker section under Concordium node distributions.